build.gradle

```diff
@@ -97,27 +106,6 @@ task jarSources(type:Jar){
     classifier = 'source'
 }
 
-checkstyle {
-    toolVersion = '8.7'
-    configFile = file('./xbotcheckstyle.xml')
-}
-
-task checkstyle(type: Checkstyle) {
-    source 'src/main/java'
-    configProperties.basedir = file('.')
-    
-    // Empty classpath
-    classpath = files()
-}
-
-checkstyleMain{
```

Comment:
having these in here caused errors so I removed these 2, there might be a better way to set the checkstyle logging threshold